qiskit_metal/analyses/core/__init__.py

```diff
@@ -2,7 +2,7 @@
 
 # This code is part of Qiskit.
 #
-# (C) Copyright IBM 2017, 2021.
+# (C) Copyright IBM 2017, 2020.
```

Comment:
Should this be 2021?

Reply:
Added the fix in #642 to not reset the approvals on this issue